Modules/booking/View/Bookingdefault/addreservationAction.php

```diff
@@ -7,6 +7,19 @@
 
     <?php echo $form->htmlOpen() ?>
     <?php echo $form->getHtml($lang, false) ?>
+    <script type="module">
+        import {DynamicForms} from '/externals/pfm/dynamics/dynamicForms.js';
+        let dynamicForms = new DynamicForms();
+        let spaceId = <?php echo $id_space?>;
+        let sourceId = "recipient_id";
+        let targets = [
+            {
+                elementId: "responsible_id",
```

Comment:
Source and target identical?

Reply:
euh... "recipient" and "responsible" are not identical :)

Reply:
Bad reading.... i should rest